src/useDebouncedRef/useDebouncedRef.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { afterEach, vi, expect, describe, it } from 'vitest'
+import { ref } from 'vue'
+import { useDebouncedRef } from '@/useDebouncedRef/useDebouncedRef'
+
+afterEach(() => {
+  vi.useRealTimers()
+})
+
+describe('useDebouncedRef', () => {
+
+  it('it syncs the initial value', () => {
+    const input = ref(1)
+    const output = useDebouncedRef(input, 100)
+
+    expect(output.value).toBe(1)
+  })
+
+  it('it debounces updates to the value', async () => {
+    vi.useFakeTimers()
+    const input = ref(1)
+    const output = useDebouncedRef(input, 100)
+    input.value = 2
+
+    await vi.runAllTimersAsync()
+
+    expect(output.value).toBe(2)
```

Comment:
should there be some parallel test that verifies that the value is NOT changed until after the timers run?